deploy/crds/all_cstor_crds.yaml

```diff
@@ -2793,28 +2793,261 @@ status:
   conditions: []
   storedVersions: []
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
-  # name must match the spec fields below, and be in the form: <plural>.<group>
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.0
+  creationTimestamp: null
   name: upgradetasks.openebs.io
 spec:
-  # group name to use for REST API: /apis/<group>/<version>
   group: openebs.io
-  # version name to use for REST API: /apis/<group>/<version>
-  version: v1alpha1
-  # either Namespaced or Cluster
-  scope: Namespaced
   names:
-    # plural name to be used in the URL: /apis/<group>/<version>/<plural>
+    kind: UpgradeTask
+    listKind: UpgradeTaskList
     plural: upgradetasks
-    # singular name to be used as an alias on the CLI and for display
     singular: upgradetask
-    # kind is normally the CamelCased singular type. Your resource manifests use this.
-    kind: UpgradeTask
-    # shortNames allow shorter string to match your resource on the CLI
-    shortNames:
-    - utask
```

Comment:
Doesn't the v1 version have the short name?

Reply:
It's not required as this CR is not documented anywhere and only used by high-level operators for automated upgrades.